### PR TITLE
#2958 Reverted the Espresso version to 3.3.0

### LINKED ISSUE #2958: UI fails to AutoConnect in the CI
<del> Using 1.4.0 libraries and </del>  In espresso 3.4.0, I occasionally run into the following RuntimeException with a simple call to getInstance(InstrumentationRegistry.getInstrumentation())
>org.kiwix.kiwixmobile.intro.IntroFragmentTest > viewIsSwipeableAndNavigatesToMain[test(AVD) - 5.0.2] FAILED 
	java.lang.RuntimeException: Error while connecting UiAutomation
	at android.app.UiAutomation.connect(UiAutomation.java:210)

No tests particularly at this point. It fails sometimes on CI emulation at the start in both API 21 & API 30.
**More Context:**
https://issuetracker.google.com/issues/233914320

### PR DESCRIPTION
**Pull request by gouri-panda**
Fixes #2958

### CODE REVIEW

**Comment by gouri-panda** [comment]
Blocked by #2985

**Comment by MohitMaliFtechiz** [comment]
hi @gouri-panda ,

is test cases are failing due to blocked PR.

**Comment by gouri-panda** [comment]
@MohitMaliFtechiz Yes.

File: buildSrc/src/main/kotlin/Versions.kt [hunk 1]
```diff
@@ -16,7 +16,7 @@ object Versions {
 
   const val org_jetbrains_kotlinx_kotlinx_coroutines: String = "1.4.1"
 
-  const val androidx_test_espresso: String = "3.4.0"
```
Comment (MohitMaliFtechiz) [inline buildSrc/src/main/kotlin/Versions.kt:19]: hi @gouri-panda , this is required for android 13 test failure , please have a look at https://github.com/kiwix/kiwix-android/commit/f69aa3521a5dc6f214ef51513a4f55cd5bb1ace8

**Review by MohitMaliFtechiz — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/kotlin/Versions.kt (hunk 1 above)
Comment (gouri-panda) [reply buildSrc/src/main/kotlin/Versions.kt:19]: @MohitMaliFtechiz Thanks for the review. @kelson42 What should we do? Should we wait for an alternative solution?

Comment (kelson42) [reply buildSrc/src/main/kotlin/Versions.kt:19]: Nonclue, but if this PR is a dead end, we shoukd close it.

**Pull request closed without merging**